src/base/_bootstrap.scss

```diff
@@ -3,5 +3,14 @@
 $icon-font-path: "#{$wb-core-path}/fonts/";
 
 
+/* TODO: Remove this block when including bootstrap 3.3.5 */
```

Comment:
They include this block for us in the latest version while they do not in the one we use right now